### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by AarthiAlagammai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,24 +11,24 @@ natural language processing.
...
 Deep learning, also known as hierarchical learning or deep structured learning, 
 is a type of machine learning that uses a layered algorithmic architecture to analyze data. 
-Unlike other types of machine learning, deep learning has the added benefit of being able to make decisions with 
-significantly less involvement of human trainers. While basic machine learning requires a programmer to identify whether a conclusion 
+Unlike other types of machine learning, deep learning has an added advantage of being able to make decisions with 
+significantly less human intervention. While basic machine learning requires a programmer to identify whether a conclusion 
 is correct or not, deep learning can gauge the accuracy of its answers on its own due to the nature of its multi-layered structure. 
 The emergence of modern frameworks like PyTorch, has also made preprocessing of data more convenient. 
-Many of the filtering and normalization tasks that would have been completed manually by human programmers when 
+Many of the filtering and normalization tasks that would involve a lot of  human programmers when
```
Comment (AarthiAlagammai) [inline README.md]: Many of the filtering and normalization tasks  would involve a lot manual efforts while using other machine learning techniques, are taken up automatically.

**Comment by AarthiAlagammai** [comment]
> I think the emphasis here was on the "manual" work, instead of the number of programmers involved.

Many of the filtering and normalization tasks would involve a lot manual efforts while using other machine learning techniques, are taken up automatically.

File: README.md [hunk 2]
```diff
@@ -64,12 +64,12 @@ Evidently, there is great potential for the future of healthcare with smart tool
 ### Potential Benefits Of Deep Learning in Healthcare?
 Among the several benefits artificial intelligence promises to bring to healthcare, these are worth highlighting.
   * **More affordable treatment:** Diagnosis is faster and more accurate with automation; doctors will be able 
-  to recommend the right treatments to patients or intervene appropriately before illnesses aggravate to 
+  to recommend the right treatments to patients or intervene appropriately before illnesses aggravates which
```
Comment (AarthiAlagammai) [inline README.md]: to recommend the right treatments to patients or intervene appropriately before illnesses aggravate leading to more expensive treatment options.

Comment (ayivima) [reply README.md]: to recommend the right treatments to patients or intervene appropriately before illnesses aggravate, leading to more expensive treatment options.

**Comment by AarthiAlagammai** [comment]
> I think we should maintain "for too long". The original intent is to emphasise how long these data have not been put to good use.

Too long doesnt seem professional

**Review by AarthiAlagammai — COMMENTED** [review]
Minor edits made as suggested by victor

**Review by ayivima — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -11,24 +11,24 @@ natural language processing.
 
 ## Abstract
 Chest X-Rays are considerably reliable radiobiological imprints of patients, which are widely used to diagnose an array 
-of common diseases affecting organs within the chest. For too long, vast accumulations of image data and their 
+of common diseases affecting organs within the chest. Over a period, vast accumulations of image data and their
```
Comment (ayivima) [inline README.md:14]: "Over a period" doesn't preserve the original intent. An alternative would be better, else we should maintain "for too long".

**Pull request merged**